grayskull/pypi/pypi.py

```diff
@@ -59,21 +58,49 @@ def __init__(
         self["build"]["script"] = "<{ PYTHON }} -m pip install . -vv"
 
     @staticmethod
-    def _download_sdist_pkg(sdist_url: str, dest: str):
+    def _get_latest_version_of_github_repo(git_url: str) -> str:
+        """get the latest version of the github repository using github api
+        """
+        latest_version_api_url = git_url.replace("www.github.com", "api.github.com/repos", 1) + "/releases/latest"
```

Comment:
This can end up being a bit fragile if the URL is weird. We'll look into a different approach :)